client/hotrod-client/pom.xml

```diff
@@ -571,6 +571,22 @@
                </execution>
             </executions>
          </plugin>
+         <plugin>
+            <groupId>org.eclipse.transformer</groupId>
+            <artifactId>org.eclipse.transformer.maven</artifactId>
+            <executions>
```

Comment:
I would move the plugin execution to parent so you don't have to copy it in 3 different modules.

Reply:
Yeah, that is no bueno, creates the jar even if there are no classes with javax imports.

Reply:
Aha, thanks. I will do that.

Reply:
I have updated it to use pluginDependencies which reduces the other poms to a few lines instead. I think this is much better.